types/pdf2image/index.d.ts

```diff
@@ -0,0 +1,53 @@
+// Type definitions for pdf2image 1.2
+// Project: https://bitbucket.org/RicardoCacheira/pdf2image#readme
+// Definitions by: taoqf <https://github.com/taoqf>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+// TypeScript Version: 2.9
+
+/**
+ * A token starts with the character '%'
+ *
+ * d - the page number, if the first page of the PDF is page 1
+ * D - the page number, if the first page of the PDF is page 0
+ * i - the processed page number, if the first processed page is page 1
+ * I - the processed page number, if the first processed page is page 0
+ * t - the total number of pages in the pdf
+ * T - the total of processed pages
+ * s - the name of the pdf file
+ * p - the path of the pdf file
+ * % - the character '%'
+ * {...} - a custom piece of code where all of the above values can be used
+ */
+
+/// <reference types="node" />
+
+export type OutputFormat = ((page_num: number, page_index: number, totalPagesProcessed: number, totalPDFPages: number, name: string, path: string, vm: typeof import ('vm')) => string) | string;
```

Comment:
`page_num`, and `page_index` name is incorrect, should be `pageNum`, `pageIndex`  based on the documentation.